PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -189,8 +190,8 @@ Usage: ${script_name} [options]
   --template2mmmask=<file path>       Brain mask MNI 2mm Template
   --brainsize=<size value>            Brain size estimate in mm, 150 for humans
   --fnirtconfig=<file path>           FNIRT 2mm T1w Configuration file
-  --fmapmag=<file path>               Siemens Gradient Echo Fieldmap magnitude file
-  --fmapphase=<file path>             Siemens Gradient Echo Fieldmap phase file
+  --fmapmag=<file path>               Siemens/Philips Gradient Echo Fieldmap magnitude file
```

Comment:
Why not new arguments?  Is a Philips fieldmap formatted the same as a Siemens fieldmap?

Reply:
Yes, both Philips and Siemens field map scans consist of two separate MRI images, one for the magnitude data and the other for phase data. In contrast, General Electric field map scans have magnitude and phase data stored in two volumes of a single image.